js/sign/src/wbn-sign.ts

```diff
@@ -1,8 +1,6 @@
 export {
-  IntegrityBlockSigner,
   IntegrityBlock,
-  WebBundleId,
-  parsePemKey,
-  readPassphrase,
-  getRawPublicKey,
-} from './integrityblock.js';
+  IntegrityBlockSigner,
+} from './signers/integrityblock.js';
+export { WebBundleId } from './web-bundle-id.js';
+export * from './utils/utils.js';
```

Comment:
I feel like exporting all utils, even future ones, is a bit bold, since all of them will become part of the public API of this package. I would only export the minimum amount that is necessary, and probably even try to reduce the amount of utils that are exported. 
For now, this means explicitly listing all the things you want to export instead of using `*`.

Reply:
We will get rid of the utils exports after the interfaces are introduced so I dont think it's a problem, but will change this to export them by their names.